meta-lxatac-bsp/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -3,6 +3,11 @@ RRECOMMENDS:${PN}:append = "less"
 PACKAGECONFIG:append = "lz4 coredump elfutils"
 PACKAGECONFIG:remove = "networkd"
 
+# Re-enable LLMNR (but not mDNS, which is handled by avahi),
+# which is disabled by the upstream recipe when the zeroconf DISTRO_FEATURE
+# is set.
+EXTRA_OEMESON:remove = "-Ddefault-llmnr=no"
```

Comment:
Do we actually use 'zeroconf'?

Reply:
I use LLMNR in my setup to resolve LXA TACs in my local network (and other may as well without noticing it by using e.g. `ssh root@lxatac-00010` and having it just work), but others may use mDNS (`ssh root@lxatac-00010.local`), which means zeroconf.

`systemd-resolved` can do both (with some drawbacks on the mDNS front though, when it comes to service discovery), while avahi (which is enabled by the zeroconf `DISTRO_FEATURE`) only does mDNS but has the better D-Bus interface.

Right now we do not use any more advanced zeroconf features, like announcing services or detecting other TACs in the network, which means `systemd-resolved` would be perfectly fine for us. Because of that I initially threw out avahi support and used resolved for both.

But @jluebbe (rightfully) objected, because there are plans to do service announcements and discovery in the `tacd` in the future, which avahi currently does better.